build.gradle

```diff
@@ -613,6 +622,47 @@ Bundle-Version: $version
     }
 }
 
+project (':quasar-kotlin') {
+
+    apply plugin: 'kotlin'
+
+    ext.moduleName = 'co.paralleluniverse.quasar.kotlin'
+
+    configurations {
+        all {
+            resolutionStrategy {
+                // Force kotlin library versions to those in config rather than gradle rules.
+                force "org.jetbrains.kotlin:kotlin-stdlib-common:$kotlinVersion"
+                force "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
+                force "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
+                // Have to force kotlinx to use 1.3.9 for Kotlin 1.4.
+                // force "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlinxVersion"
```

Comment:
Are coroutines part of Kotlin 1.4 properly yet? Perhaps you don't need this dependency any more?